Tasks/AzureCloudPowerShellDeployment/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -25,5 +25,11 @@
   "loc.input.label.NewServiceAdditionalArguments": "Additional Arguments",
   "loc.input.help.NewServiceAdditionalArguments": "Pass in additional arguments while creating a brand new service. These will be passed on to `New-AzureService` commandlet. Eg: `-Label 'MyTestService'`",
   "loc.input.label.NewServiceAffinityGroup": "AffinityGroup",
-  "loc.input.help.NewServiceAffinityGroup": "While creating new service, this affinity group will be considered instead of using service location."
+  "loc.input.help.NewServiceAffinityGroup": "While creating new service, this affinity group will be considered instead of using service location.",
+  "loc.messages.Couldnotgettheprimarystoragekeyforstorageaccount0Unabletoapplyanydiagnosticsextensions": "Could not get the primary storage key for storage account '{0}'. Unable to apply any diagnostics extensions.",
+  "loc.messages.Couldnotgettheprimarystoragekeyforthepublicconfigstorageaccount0Unabletoapplyanydiagnosticsextensions": "Could not get the primary storage key for the public config storage account '{0}'. Unable to apply any diagnostics extensions.",
+  "loc.messages.Applyinganyconfigureddiagnosticsextensions": "Applying any configured diagnostics extensions.",
+  "loc.messages.0couldnotbeparsedintopartsforregisteringdiagnosticsextensions": "'{0}' could not be parsed into parts for registering diagnostics extensions.",
```

Comment:
I'm not sure starting with a number is legal. You might need to start with `_` so this doesnt break the loc process. Can you switch to a leading `_`? I'll followup on the loc process and add a unit test if required.